cob_bringup_sim/scripts/spawn_object.py

```diff
@@ -59,6 +59,7 @@
 import sys
 import os
 
+import roslib
```

Comment:
Hasn't this been done in https://github.com/ipa320/cob_simulation/commit/ec32055b7d057d70994e21752a89fcc2593944b1 already?
Please pull ipa320/indigo_dev or rebase...